tasks/main.yml

```diff
@@ -68,7 +68,7 @@
   tags: ['disk']
 
 - name: "Get UUID for partition"
-  command: blkid -s UUID -o value "{{ item.disk }}1"
+  command: blkid -s UUID -o value "{{ item.part | default(item.disk + '1') }}"
```

Comment:
Just for consistency, do you want to double quote the `1` (e.g. `+ "1"`) or single quote the others?

The rest LGTM. 👍